packages/p2p/src/components/payment-method-card/payment-method-card.tsx

```diff
@@ -2,11 +2,27 @@ import React from 'react';
 import classNames from 'classnames';
 import { Checkbox, Dropdown, Icon, Text } from '@deriv/components';
 import { isEmptyObject } from '@deriv/shared';
+import { useStores } from 'Stores';
 import { localize } from 'Components/i18next';
 import { useModalManagerContext } from 'Components/modal-manager/modal-manager-context';
 import PaymentMethodIcon from 'Components/payment-method-icon';
-import { useStores } from 'Stores';
-import PropTypes from 'prop-types';
+import { TPaymentMethod } from 'Types/my-profile.types';
+
+type TPaymentMethodCardProps = {
+    add_payment_method?: string;
+    disabled?: boolean;
+    is_add?: boolean;
+    is_vertical_ellipsis_visible?: boolean;
+    label?: string | undefined;
```

Comment:
```suggestion
    label?: string;
```

🤔

Reply:
ah i defined it as that cause it was the default value but i dont think we need to define it as undefined